applications/callflow/src/cf_endpoint.erl

```diff
@@ -1197,9 +1197,26 @@ generate_sip_headers(Endpoint, Acc, Call) ->
     HeaderFuns = [fun maybe_add_sip_headers/1
                   ,fun(J) -> maybe_add_alert_info(J, Endpoint, Inception) end
                   ,fun(J) -> maybe_add_aor(J, Endpoint, Call) end
+                  ,fun(J) -> maybe_add_diversion(J, Call, Inception, Endpoint) end
```

Comment:
My sensibilities are screaming about arg order here! :)

Could we do `maybe_add_diversion(J, Endpoint, Inception, Call)` since `Endpoint` is typically the second arg in these funs?